test/common-utils/scenarios.json

```
@@ -259,5 +259,22 @@
         "features": {
             "common-utils": {}
         }
+    },
+    "terminal-title-on-xterm": {
+        "image": "node",
+        "features": {
+            "common-utils": {}
+        },
+        "containerEnv": {
+            "TERM": "xterm"
+        }
+    },
+    "no-terminal-title-by-default": {
+        "image": "node",
+        "features": {
+            "common-utils": {}
+        },
+        "containerEnv": {
+        }
```

```suggestion
        }
```

Let's remove as we are not adding any env variables?